code/datums/mutations/antenna.dm

```diff
@@ -28,7 +28,7 @@
 	if(..())
 		return
 	if(linked_radio)
-		linked_radio.Destroy()
+		QDEL_NULL(linked_radio)
```

Comment:
```suggestion
	QDEL_NULL(linked_radio)
```

Reply:
you can make multi line suggestions you know

Reply:
you cant for lines that were deleted, try it.

Reply:
fair enough